acceptance/lib/helper.rb

```diff
@@ -196,6 +196,22 @@ def get_defaults_var(host, varname)
     stdout
   end
 
+  def hup(host = master, timeout = 30)
```

Comment:
Maybe name it hup_server()? Bare hup() suggests it can be used to HUP other things.